db/schema.rb

```diff
@@ -162,9 +162,9 @@
     t.string "why_dont_you_know_la"
     t.integer "unitletas"
     t.integer "builtype"
+    t.datetime "property_void_date"
```

Comment:
Doesn't look like this should really be here since there's no migration?

Reply:
Yeah, I'll move it back